modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -71,5 +71,24 @@ map[name:custom-certs-default]
 +
 The certificate secret name should match the value used to update the CR.
 
+Your Ingress Controller now uses the custom certificate, but it is not trusted
+by the web console. You must expose your CA certificate so the Ingress
+Controller can communicate with all routes.
+
+. Configure the CA certificate as the cluster proxy certificate:
++
+----
+$ oc -n openshift-config create configmap custom-ca --from-file=ca-bundle.crt=example-ca.crt
```

Comment:
A note should be added that if proxy is already has a `trustedCA` reference, the existing ConfigMap should be updated (`$oc edit`) to include the custom certificate instead of creating a new configmap and patching proxy/cluster.